docs/install.md

```diff
@@ -75,7 +75,7 @@ installation method.
 Download the latest [.deb package][latest-release]*, and run the following:
 
 ```shell
-sudo dpkg -i <path-to-package>
+sudo gdebi <path-to-package>
```

Comment:
> Latest Ubuntu 24.04 uses Gdebi to install Debian packages

Hrmmmm. I tried to reconstruct the (missing) reference to the official Ubuntu announcement to that end, and failed. It does not look like `gdebi` is _the_ Ubuntu 24.04 way to install `.deb` packages. In fact, all information I could find on the matter suggested that `dpkg` is still installed by default, and `gdebi` is not.

In any case, there's more versions of Ubuntu, and there's quite a few Debian package-based Linux distributions that happen not to be Ubuntu. Given this lack of ubiquity, I would have expected _maybe_ to augment the existing command with a suggestion to use `gdebi` instead, on Ubuntu 24.04, but certainly not to replace the advice to use `dpkg`.

So: hrmmmm.